pymemcache/test/test_codecs.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+from unittest import TestCase
+
+from pymemcache import codecs
+import pytest
+import six
+
+try:
```

Comment:
We should import this the same way we do elsewhere in this code: `from six.moves import cPickle as pickle`

Reply:
done